src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Figure;
+import core.basesyntax.supplier.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        final int Array_Size = 5;
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[Array_Size];
+        for (int i = 0; i < figures.length; i++) {
+            if (i == 2) {
+                figures[i] = figureSupplier.getDefaultFigure();
+            } else {
+                figures[i] = figureSupplier.getRandomFigure();
+            }
+        }
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/8d21f80d-b4d3-46d7-bd96-44a965d671c0)

Reply:
if an array contains 6 elements, the first half for example(square, triangle, rectangle) should be with random parameters, and in other half should be a white circle with default parameters. should other figures in the second half be with default parameters or with random?

Reply:
@Rommelua if an array contains 6 elements, the first half for example(square, triangle, rectangle) should be with random parameters, and in other half should be a white circle with default parameters. should other figures in the second half be with default parameters or with random?